ppdet/data/source/widerface.py

```diff
@@ -62,30 +64,23 @@ def load_roidb_and_cname2cid(self):
 
         records = []
         ct = 0
-        file_lists = _load_file_list(txt_file)
+        file_lists = self._load_file_list(txt_file)
         cname2cid = widerface_label(self.with_background)
 
         for item in file_lists:
             im_fname = item[0]
             im_id = np.array([ct])
-            gt_bbox = np.zeros((len(item) - 2, 4), dtype=np.float32)
-            gt_class = np.ones((len(item) - 2, 1), dtype=np.int32)
+            gt_bbox = np.zeros((len(item) - 1, 4), dtype=np.float32)
```

Comment:
why change the size of gt_bbox?

Reply:
Cleaned the code in `_load_file_list`, so dimension of gt_bbox decreased.